File: Digital Business Enablement/buy.it-api/src/main/java/br/com/fiap/buy/it/dto/ProdutoDTO.java
package br.com.fiap.buy.it.dto;

import lombok.*;

import java.util.Set;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class ProdutoDTO {
    private Long id;
    private String nome;
    private String marca;
    private String cor;
    private String tamanho;
    private String material;
    private String observacao;
    private Long idDepartamento;
    private Set<Long> idsTags;
}

File: Enterprise Application Development/buyit/Dtos/ProdutoDto.cs
using System.ComponentModel.DataAnnotations;

namespace Buyit.Dtos
{
    public class ProdutoDto
    {
        public long? Id { get; set; }

        [Required(ErrorMessage = "O campo nome não pode estar vazio.")]
        public string? Nome { get; set; }

        public string? Marca { get; set; }
        public string? Cor { get; set; }
        public string? Tamanho { get; set; }
        public string? Material { get; set; }
        public string? Observacao { get; set; }
        public long? IdDepartamento { get; set; }

        public List<long>? IdsTags { get; set; }
    }
}